configuration/display_rotation.md

```diff
@@ -0,0 +1,35 @@
+## How to rotate your display
+
+The options to rotate the display of your Raspberry Pi depend on which display driver software it is running, which may also depend on which Raspberry Pi you are using. 
+
+### Fake or Full KMS graphics driver (Default on Pi4)
+
+If you are running the Raspberry Pi desktop then rotation is achieved by using the `Screen Configuration Utility` from the desktop `Preferences` menu. This will bring up a graphical display of the display or displays attached. Right click on the display you wish to rotate and select the required option.
+
+It is also possible to change these settings using the command line `xrandr` option.
+
+```bash
+DISPLAY=:0 xrandr --output HDMI1 --rotate left 
+DISPLAY=:0 xrandr --output HDMI1 --rotate inverted
+DISPLAY=:0 xrandr --output HDMI1 --rotate normal
```

Comment:
Probably worth explaining what "rotate inverted" means? Is there also a "rotate right"? And (for completeness) is it worth listing the available `--output` options (e.g. `HDMI0`)? Is there also an option for rotating the composite video output? And the DSI LCD?
(I assume there's no HDMI1 when using the KMS driver on Pi3?)

Reply:
I still think a little bit of clarification around the usage of the `--output` option would be useful here.

Reply:
Added, along with some text on the reflect option which does work.